pkg/controller/publishingstrategy/publishingstrategy_controller.go

```diff
@@ -358,6 +358,13 @@ func (r *ReconcilePublishingStrategy) deleteIngressWithAnnotation(appIngressList
 		if _, ok := ingress.Annotations["Owner"]; !ok {
 			continue
 		}
+		// if ingress is default, continue as we are only looking at non-default ingresses
```

Comment:
```suggestion
		// if ingress is default, skip it since we are only looking at non-default ingresses
```

`continue` is a Go keyword, but also could mean "continue with the flow of code" as in, something desired. So minor language suggestion here.